Firestore/core/src/firebase/firestore/auth/token.h

```diff
@@ -58,26 +58,14 @@ class Token {
     return user_;
   }
 
-  /**
-   * Whether the token is a valid one.
-   *
-   * ## Portability notes: Invalid token is the equivalent of nil in the iOS
-   * token implementation. We use value instead of pointer for Token instance in
-   * the C++ migration.
-   */
-  bool is_valid() const {
-    return is_valid_;
-  }
-
-  /** Returns an invalid token. */
-  static const Token& Invalid();
+  /** Returns a token for an unauthenticated user. */
```

Comment:
Please get in the habit of double-checking whether or not there are equivalents on the other platforms and either:
  * avoid the new concept,
  * port use concepts over,
  * make PORTABILITY NOTES indicating the deviation and why, or
  * file bugs to sort this out as a follow-up

There is no equivalent concept on Android and the production code path in the credential provider doesn't actually use this to create the unauthenticated token.

Reply:
Done.